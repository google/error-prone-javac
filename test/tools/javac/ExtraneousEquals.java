/*
 * @test /nodynamiccopyright/
 * @bug 5019614
 * @summary variance prototype syntax leftover
 *
 * @compile/fail/ref=ExtraneousEquals.out -XDrawDiagnostics ExtraneousEquals.java
 */

public class ExtraneousEquals {
  int[] foo = new int[=] { 1, 2, 3 };
}
