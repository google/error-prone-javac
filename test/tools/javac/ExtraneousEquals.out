ExtraneousEquals.java:10:23: compiler.err.illegal.start.of.expr
ExtraneousEquals.java:10:24: compiler.err.illegal.start.of.expr
ExtraneousEquals.java:10:25: compiler.err.expected: ';'
ExtraneousEquals.java:10:28: compiler.err.not.stmt
ExtraneousEquals.java:10:29: compiler.err.expected: ';'
5 errors
